components/vs-code-extension/code/vs-code-extension-bbe/ballerina-copilot.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Ballerina Copilot"
+description: "
+
+Ballerina Copilot seamlessly integrates AI-driven capabilities into your integration developement workflow, transforming natural language into high-quality Ballerina code. Boost your productivity with powerful features designed specifically for Ballerina developers, including:
+
+-   AI-Powered Code Generation: Write complex Ballerina code effortlessly using simple, natural language prompts.
+    
+-   Automated Test Generation: Quickly create comprehensive tests for your services, ensuring reliability and robustness.
+    
+-   Ask Ballerina Documentation: Easily navigate and query Ballerina documentation directly using natural language, saving valuable time and streamlining your coding experience.
+    
+-   Intelligent Data Mapping: Simplify your integration tasks with automated data mapping, intelligently bridging data structures and types.
```

Comment:
I think the order should be
1. code gen
2. data mapping
3. ask
4. test


wdyt?

Reply:
makes sense, will change accoringly

Reply:
Either
1. code gen
2. data mapping
3. ask
4. test

or

1. code gen
2. ask
3. data mapping
4. test